openmc/opencg_compatible.py

```diff
@@ -455,37 +455,37 @@ def get_opencg_cell(openmc_cell):
         raise ValueError(msg)
 
     global OPENCG_CELLS
-    cell_id = openmc_cell._id
+    cell_id = openmc_cell.id
 
     # If this Cell was already created, use it
     if cell_id in OPENCG_CELLS:
         return OPENCG_CELLS[cell_id]
 
     # Create an OpenCG Cell to represent this OpenMC Cell
-    name = openmc_cell._name
+    name = openmc_cell.name
     opencg_cell = opencg.Cell(cell_id, name)
 
-    fill = openmc_cell._fill
+    fill = openmc_cell.fill
 
-    if (openmc_cell._type == 'normal'):
-        opencg_cell.setFill(get_opencg_material(fill))
-    elif (openmc_cell._type == 'fill'):
-        opencg_cell.setFill(get_opencg_universe(fill))
+    if (openmc_cell.fill_type == 'material'):
+        opencg_cell.fill = get_opencg_material(fill)
+    elif (openmc_cell.fill_type == 'universe'):
```

Comment:
Shouldn't this be `openmc_cell.type`? And actually, looking at the source, it looks like there's a bug in that property (it returns `self._fill`. Can you fix that?